src/ServiceInsight/ServiceControl/IServiceControl.cs

```diff
@@ -34,5 +34,8 @@ public class ServiceControlHeaders
     {
         public const string ParticularVersion = "X-Particular-Version";
         public const string TotalCount = "Total-Count";
+        public const string Link = "Link";
+        public const string PageSize = "Page-Size";
+        public const string PageNumber = "Page-Number";
```

Comment:
We never use this, and instead we initialise the page number to 1 locally and then increment and decrement it locally when navigating. Is there any reason we don't read this from the response instead?